moviepy/audio/fx/multiply_volume.py

```diff
@@ -1,22 +1,68 @@
-from moviepy.decorators import audio_video_fx
+import numpy as np
+
+from moviepy.decorators import audio_video_fx, convert_parameter_to_seconds
+
+
+def _multiply_volume_in_range(factor, start, end, nchannels):
+    def factors_filter(factor, t):
+        return np.array([factor if start <= t_ <= end else 1 for t_ in t])
+
+    def multiply_stereo_volume(get_frame, t):
+        return np.multiply(
+            get_frame(t),
+            np.array([factors_filter(factor, t) for _ in range(nchannels)]).T,
+        )
+
+    def multiply_mono_volume(get_frame, t):
+        return np.multiply(get_frame(t), factors_filter(factor, t))
+
+    return multiply_mono_volume if nchannels == 1 else multiply_stereo_volume
 
 
 @audio_video_fx
-def multiply_volume(clip, factor):
+@convert_parameter_to_seconds(["start", "end"])
+def multiply_volume(clip, factor, start=None, end=None):
```

Comment:
I'm wondering about what would be the correct name for these new arguments `start` and `end`. In #1170 I've seen that you've renamed `t_start` and `t_end` by `start_time` and `end_time` @tburrows13, but for example `Clip` uses `start` and `end` property names instead of `start_time` and `end_time` (also the correspondient method `with_start`). So, this shouldn't be normalized in the codebase? What do you think about `start` and `end` argument names, it would be better to use `start_time` and `end_time` here?

Reply:
Sorry, I missed this message earlier! Yeah, consistency is good. I'm not sure which is better though. I'd lean towards `start_time` instead of `start` but I haven't investigated to see which is used where, and it could be that in some usages `start` makes more sense.

Reply:
Thank you for the feedback, I would opt for `start_time` in this case to make it explicit :+1: